packages/angular/src/editor/editor.component.ts

```diff
@@ -363,7 +368,7 @@ export class EditorComponent extends RootComponent implements OnInit, TriggerInt
 
     // Froala Configuration
     froalaConfig: LooseObject = {
-        // key: 'LGnD1KNZf1CPBYCAZB-8F3UDSLLSG1VFf1A3C2==',
+        key: 'DUA2yE1F1A1A4B2C2pZGCTRSAPJWTLPLZHTQQe1JGZxC4B3A3C2B5B1A1E4G1B3==',
```

Comment:
is this key something we're allowed to publicly share outside the Enterprise license?

Reply:
I could hide it in the HTML to be more safe, but there is the issue that anyone can still view it with a `view-source`.

Reply:
I don't think even the source could be prevented unless froala enhances their own security